KubeArmor/core/kubeArmor.go

```diff
@@ -448,6 +473,44 @@ func KubeArmor() {
 
 	// == //
 
+	if !dm.K8sEnabled && cfg.GlobalCfg.Policy {
+		// check if the CRI socket set while executing kubearmor exists
+		if cfg.GlobalCfg.CRISocket != "" {
+			trimmedSocket := strings.TrimPrefix(cfg.GlobalCfg.CRISocket, "unix://")
```

Comment:
do you know when we use "trimmedSocket"?

Reply:
To get the sock file path from the -crioSocket flag and to check whether the sock file exist or not?